taichi/aot/module_builder.h

```diff
@@ -37,6 +38,10 @@ class AotModuleBuilder {
   virtual void dump(const std::string &output_dir,
                     const std::string &filename) const = 0;
 
+  void dump_graph(std::string output_dir) const;
```

Comment:
OOC, why we need a separate `dump_graph`, instead of dumping everything in `dump`?

Reply:
@k-ye Since `dump_graph` is backend independent so I wanted to keep it in the base class. Note it's called in the `dump` in every backend indeed we didn't call them separately.

Reply:
doesnt need to be public?

Reply:
aha nice catch! it was a legacy public api used in the first prototype. Removed in #5024! :D